src/docfx/localization/LocalizationConvention.cs

```diff
@@ -122,6 +120,53 @@ public static bool TryGetContributionBranch(string branch, out string contributi
             return false;
         }
 
+        public static (Error error, string content, Document file) TryResolveFromHistory(Docset docset, string pathToDocset)
+        {
+            // try to resolve from source repo's git history
+            // todo: support code snippet
```

Comment:
code snippet should be supported out of the box because it is also calling ResolveContent, we just need a test.

Reply:
it's related to the content type, i will do it in another pr

---
In reply to: [237189150](https://github.com/dotnet/docfx/pull/3677#discussion_r237189150) [](ancestors = 237189150)